src/cljs/rems/roles.cljs

```diff
@@ -0,0 +1,24 @@
+(ns rems.roles
+  (:require [clojure.set :as set]))
+
+(defn- has-some? [expected-roles actual-roles]
+  (not (empty? (set/intersection expected-roles actual-roles))))
+
+(defn is-logged-in? [roles]
+  (has-some? #{:logged-in} roles))
+
+(defn is-applicant? [roles]
+  (has-some? #{:applicant} roles))
+
+(defn is-handler? [roles]
```

Comment:
This function is not named well where the `handler` word is now reserved for the superuser and this should be something totally different. Possibly you could use `is-actor?` because e.g. the page is named actions but that is also overloaded in the event terminology.

Reply:
How about "reviewer"? After removing the old workflow, that word should be free. The handler and others are all reviewing the application. [Definitions of review](https://www.dictionary.com/browse/review) include "to go over (lessons, studies, work, etc.) in review", "to view, look at, or look over again", "to inspect, especially formally or officially". "Inspector" could be used as a synonym.

The name "actions" is also overloaded. The buttons in the application page are called actions and some of the actions are done by the applicant. Calling it the review page would be more descriptive.

Reply:
I think for our the users the review means something else so I would not be so eager to use it either. They are used to so-called review and third party review functionality. Have to think about this.